bpo-43908: Make array.array type immutable (#25696)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2021-04-26-20-59-17.bpo-43908.-COW4-.rst

> Apply :const:`Py_TPFLAGS_IMMUTABLETYPE` flag to :class:`array.array`. Patch by

Reviewer: ```suggestion
Make the :class:`array.array` type immutable. Patch by
```
